launchable/commands/record/session.py

```diff
@@ -11,6 +13,15 @@
 
 LAUNCHABLE_SESSION_DIR_KEY = 'LAUNCHABLE_SESSION_DIR'
 
+JENKINS_URL_KEY = 'JENKINS_URL'
+JENKINS_BUILD_URL_KEY = 'BUILD_URL'
+GITHUB_ACTION_KEY = 'GITHUB_ACTION'
```

Comment:
[question]
Is it ok we have `GITHUB ACTION` here and `GITHUB ACTIONS` below?
https://github.com/launchableinc/cli/pull/436/files#diff-83169579c4ecf3f02d6c5587f3d60e8727f3799b09f8d97a1f75db4dd2f688bdR6

Official name is `GitHub Actions`.
https://github.com/features/actions

Reply:
It seems to be that `GITHUB_ACTION` and `GITHUB_ACTIONS` both exist. https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables

In our case that only confirms a job is running on Github Actions or not. So I think we should use `GITHUB_ACTIONS`